packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -61,6 +61,9 @@ class FocusAttachment {
   void detach() {
     assert(_node != null);
     if (isAttached) {
+      if (_node.hasPrimaryFocus) {
+       _node.unfocus();
```

Comment:
Minor nit, missing a space of indentation?

Reply:
Oooh, good eye.  Yes, I'll fix that.